tests/contrib/langchain/test_langchain_community.py

```diff
@@ -1274,3 +1274,54 @@ def test_faiss_vectorstore_retrieval(langchain_community, langchain_openai, requ
             retriever = faiss.as_retriever()
         with request_vcr.use_cassette("openai_retrieval_embedding.yaml"):
             retriever.invoke("What was the message of the last test query?")
+
+
+@pytest.mark.snapshot
+def test_base_tool_invoke(langchain, langchain_core, request_vcr):
+    """
+    Test that invoking a tool with langchain will
+    result in a 1-span trace with a tool span.
+    """
+    from math import pi
+
+    cassette_name = "langchain_tool_invoke.yaml"
+    with request_vcr.use_cassette(cassette_name):
```

Comment:
AFAIK tools can be called without a network connection. Why do we need a cassette for this?

Reply:
Yes gotcha ! Was thinking of adding the tool to an LLM instead opted for a simple test. I did remove it in an unpushed commit. Thanks !
I still left the option to call invoke with a cassette in llmobs side for any future agent test to be made.